cypress/e2e/tests/pages/generic/login.spec.ts

```diff
@@ -6,14 +6,19 @@ const successStatusCode = 200;
 
 describe('Local authentication', { tags: ['@generic', '@adminUser', '@standardUser'] }, () => {
   it('Confirm correct number of settings requests made', () => {
-    cy.intercept('GET', '/v1/management.cattle.io.settings?exclude=metadata.managedFields').as('settingsReq');
+    cy.intercept(
```

Comment:
you might have some alternative linting rules making these changes. might be worth double checking your local env. not a blocker for this PR, but could introduce churn later on